babel/dates.py

```diff
@@ -1282,6 +1286,14 @@ def format_timezone(self, char, num):
                 return get_timezone_name(self.value.tzinfo, width,
                                          uncommon=True, locale=self.locale)
             return get_timezone_location(self.value.tzinfo, locale=self.locale)
+        # Included additional elif condition to add support for 'X' in timezone format
+        elif char in ['x','X']:
+            if num==1:
+                return get_timezone_gmt(self.value, width='short', locale=self.locale)[:-2]
+            elif num in [2,4]:
```

Comment:
Use a tuple `num in (2, 4)` or a simple `or` statement here (as in my comment above).